src/tiledimage.js

```diff
@@ -1784,20 +1765,6 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
         if (tile.loaded && tile.opacity === 1){
             this._setCoverage( this.coverage, level, x, y, true );
         }
-        if ( haveDrawn && !drawTile ) {
```

Comment:
Yes, good point! I was focusing on `haveDrawn` here and not paying attention to the fact `!drawTile` would never happen.

Reply:
> Yes, good point! I was focusing on `haveDrawn` here and not paying attention to the fact `!drawTile` would never happen.

Yeah, it is convoluted! Once I deleted this `if()` statement, that's when the argument `haveDrawn` (which comes from `drawLevel` in the calling function) became unused.

Are you in agreement that this can be dropped, and that the upstream `drawLevel` can therefore also be dropped?

Reply:
Yes, I agree. Thank you for confirming :)